examples/shadowmap/src/index.js

```diff
@@ -0,0 +1,138 @@
+import 'babel-polyfill';
+
+const glslify = require('glslify');
+import * as LumaGL from '../../../src/index.js';
+
+window.webGLStart = function() {
+
+  var createGLContext = LumaGL.createGLContext;
```

Comment:
If we are taking the trouble to run this file through babel (as opposed to just loading a prebuilt script) it could look nicer if we included the luma stuff through the import, as opposed to ES5 style destructuring.

import (createGLContext, Program, ... } from '../../../src';

(Note you can also drop the 'index.js' from the import.)